integration_tests/suite/test_confgen.py

```diff
@@ -0,0 +1,159 @@
+import subprocess
+import pytest
+from wazo_test_helpers import asset_launching_test_case
+import pathlib
+import sys
+from typing import List, Iterable
+
+
+INTERNAL_CONFGEND_PORT = 8669
+DEFAULT_CONFGEND_CLIENT_TIMEOUT = 10
+DEFAULT_TIMEOUT = 10
+
+@pytest.fixture(scope="class")
+def pjsip_void_conf(request):
+    with open("suite/pjsip_void.conf") as file:
+        conf = file.readlines()
+
+    request.cls.pjsip_void_conf = conf
+
+
+def normalize_lines(line_stream: Iterable[str]):
+    for line in line_stream:
+        line = line.strip()
+        if line:
+            yield line
+
+
+def lines(text: str):
+    return text.splitlines()
```

Comment:
I don't get the plus value of this helper. Why not using `splitlines()` directly in the code?
It would avoid indirection. Moreover it's only used once